src/signalo/settings.py

```diff
@@ -146,6 +146,7 @@
     # "geocity.auth.InternalTokenAuthentication",
     # ),
     "DEFAULT_PAGINATION_CLASS": "django_oapif.pagination.OapifPagination",
+    "DEFAULT_PERMISSION_CLASSES": ("rest_framework.permissions.AllowAny",),
```

Comment:
This is super-permissive, I guess it's extremely rare that one wants just anyone to be able to edit their data (basically a wiki). As discussed, let's default to `[DjangoModelPermissionsOrAnonReadOnly](https://www.django-rest-framework.org/api-guide/permissions/#djangomodelpermissionsoranonreadonly)`.

Then you can add a new "WikiModel" class with `AllowAny` permission, and "SecretModel" with `IsAdminUse`, just to demonstrate that it works (add these to the `edge_cases` app)

Reply:
> As with DjangoModelPermissions, this permission must only be applied to views that have a .queryset property or .get_queryset() method.

This condition is not met as of this PR.

Reply:
Indeed didn't think about the RootView and CollectionView (that's what you're refering to, right ?)

Still, defaulting to wiki isn't good enough. Either we could 1/ still put the DjangoModelPermissionsOrAnonReadOnly as a default, and override with AllowAny on RootView and CollectionView. Otherwise we could 2/ introduce a new setting, called `OGCAPIF_DEFAULT_PERMISSION_CLASSES` (outside of the native DRF settings dict) that we would use as a default in our decorator or even better directly on the `OAPIFDescribeModelViewSetMixin`.

IMO better to go with 1/, it's more standard, less stuff to document. The only drawback is that it would be difficult to protect RootView and CollectionView in case one doesn't want to expose them to unlogged users (for RootView, I think we always want to expose it, as it may also include login endpoint, and for collectionview see my commend below)